compiler/noirc_frontend/src/elaborator/mod.rs

```diff
@@ -1228,6 +1229,7 @@ impl<'context> Elaborator<'context> {
             self.add_generics(&trait_impl.generics);
             trait_impl.resolved_generics = self.generics.clone();
 
+            // changes here
```

Comment:
```suggestion
```
Not sure what this means. We should delete or elaborate more

Reply:
It was left over from debugging, I'll remove it